flow.js

```diff
@@ -1,6 +1,6 @@
 
 class Flow {
-  constructor(setState, history, path, prevSteps, setPrevSteps) {
+  constructor(setState, history, path, prevSteps, setPrevSteps, repairLocationNextSteps = undefined) {
```

Comment:
This is used for location, problem and best description next steps and should be renamed appropriately, i.e. 
```suggestion
  constructor(setState, history, path, prevSteps, setPrevSteps, repairTriageNextSteps = undefined) {
```

Reply:
I've renamed this. Thanks!